src/tic_tac_clojure/player.clj

```diff
@@ -0,0 +1,52 @@
+(ns tic-tac-clojure.player
+  (:gen-class)
+  (:require [tic-tac-clojure.ai :refer :all]
+            [tic-tac-clojure.board :refer :all]
+            [tic-tac-clojure.cli :refer :all]
+            [tic-tac-clojure.game-logic :refer :all]
+            [tic-tac-clojure.message-render :refer :all]))
+
+(defn get-player-mark
+  [player]
+  (:mark player))
+
+(defn is-human?
+  [player]
+  (:human? player))
+
+(defn receive-board-idx-input 
+  [board]
+  (let [selection (get-valid-num-input 0 
+                                       (- (count board) 1))]
```

Comment:
In order to avoid having three nearly identical validators for input regarding selecting a game, marking the board, and playing again, I created `get-valid-num-input`, which takes an inclusive starting and ending range and ensures that the user input is both a number and a number within that range. In this case, 0 is the lowest possible index on the board, and the 1 has to be subtracted from the length of the board in order to ensure that a number is within the inclusive range (so that 9 is acknowledged as invalid). 

I can think of a few ways to improve the clarity of the function:
1. Change the name to something like `get-valid-num-input-within-range`.
2. Instead of taking a range, the `get-valid-num-input` can take a set of numbers, and the function can check that the user's number is included in that set. 
3. Remove the responsibility to validate from `get-valid-num-input` and handle the validation in the validators namespace. 

I think that my current approach, updating the name, keeps my code dry. However, implementing the second idea wouldn't be too difficult since I can already access the indices of all the empty spaces on the board via `empty-space-indices` in the board namespace. Is it enough to change the name of the function?